src/test/kotlin/org/rust/ide/console/RsConsoleCompletionTestBase.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.console
+
+import org.rust.ProjectDescriptor
+import org.rust.RsTestBase
+import org.rust.WithStdlibRustProjectDescriptor
+import org.rust.lang.core.completion.RsCompletionTestFixture
+import org.rust.lang.core.psi.RsFile
+import org.rust.lang.core.psi.RsReplCodeFragment
+
+@ProjectDescriptor(WithStdlibRustProjectDescriptor::class)
+abstract class RsConsoleCompletionTestBase : RsTestBase() {
+
+    private lateinit var completionFixture: RsCompletionTestFixture
+
+    override fun setUp() {
+        super.setUp()
+        completionFixture = RsCompletionTestFixture(myFixture)
+        completionFixture.setUp()
+    }
+
+    override fun tearDown() {
+        completionFixture.tearDown()
+        super.tearDown()
+    }
+
+    protected fun checkContainsCompletion(previousLines: String, lastLine: String, variants: Array<String>) {
```

Comment:
Maybe it's worth to replace `variants: Array<String>` to `vararg variants: String`?
It allows to avoid `arrayOf` in tests